README.md

```diff
@@ -1,3 +1,115 @@
-# perception_ecu_container
+# Perception ECU Container
 
-This repository provides a sample environment working with Jetson + GMSL cam + ROS2.
+This repository provides a sample ROS2 environment working on a Jetson AGX Xavier based ECU and GMSL2-compatible cameras.
```

Comment:
```suggestion
This repository provides a sample ROS2 environment for object recognition using [tensorrt_yolox](https://github.com/autowarefoundation/autoware.universe/tree/main/perception/tensorrt_yolox). It works on a Jetson AGX Xavier-based ECU and GMSL2-compatible cameras.
```

Reply:
@kaz0227 Since we'll add more applications in the future, how about writing them arouind here instead of the top level of this readme?
https://github.com/tier4/perception_ecu_container/blob/bbda94b05318e598fd91c69b485adba27dea0b82/README.md?plain=1#L90

Reply:
@aohsato I see. Let's add the comment there. Also, how about adding the comment here?
"As a sample application, the following images show object recognition results using the contents of this repository. We plan to add other applications here and release them as open-source software (OSS)."
https://github.com/tier4/perception_ecu_container/blob/1a93e75ce9f122f18d506383a5be2b999160c6c6/README.md?plain=1#L4

Reply:
@kaz0227 :cool: plz create your suggestion if you okay?

Reply:
@aohsato Created a suggestion.
https://github.com/tier4/perception_ecu_container/pull/1/files#r1058147127

Reply:
@kaz0227 Thanks for your comment. Additionally, I added some tensorrt_yolox links, please review?
[1a2b288](https://github.com/tier4/perception_ecu_container/pull/1/commits/1a2b288afd6264366f8844343430ac041861817f)